README.md

```diff
@@ -21,32 +21,41 @@ If you wish to contribute, we highly recommend following the guidelines mentione
    git clone https://github.com/your_username/devmate.git
 ```
 
-**3.** Navigate to the project directory.
+**3.** Install Dependencies.
+```
+   npm install next react react-dom
```

Comment:
```suggestion
   npm i
```

Reply:
we can add npm i to install all the dependencies!

Reply:
same for yarn and pnpm !

Reply:
Done